aip/cloud/2602.md

```diff
@@ -18,13 +20,13 @@ depending on the value of a particular parameter.
 
 For instance, suppose we're designing a new gcloud surface for managing images,
 and we need a command that formats an image as JPEG, GIF, or PNG. Regardless of
-the image format, the command will need arguments for the source and destination
-files, but otherwise depending on the format, an entirely different set of
-options will apply.  JPEG, for example, might take arguments for smoothing,
-subsampling, and DCT method, which are only relevant for JPEG images.  GIF, on
-the other hand, might take arguments for controlling animated gifs e.g.  whether
-to loop forever, and a delay time in between frames. Finally, PNG might take
-arguments for color type and bits per channel.
+the image format, the command will need arguments for the source and
+destination files, but otherwise depending on the format, an entirely different
+set of options will apply. JPEG, for example, might take arguments for
+smoothing, subsampling, and DCT method, which are only relevant for JPEG
+images. GIF, on the other hand, might take arguments for controlling animated
+gifs e.g. whether to loop forever, and a delay time in between frames. Finally,
+PNG might take arguments for color type and bits per channel.
```

Comment:
FWIW, our team has stopped running the prettier formatter on AIPs. We've found that it makes editing in GitHub easier.

Reply:
That seems like something we should continue to do on the main site for consistency, but it also seems reasonable for a fork to choose not to do it.